package/edgehog-device-runtime/Config.in

```diff
@@ -10,22 +10,21 @@ config BR2_PACKAGE_EDGEHOG_DEVICE_RUNTIME
             depends on BR2_PACKAGE_HOST_PROTOBUF_ARCH_SUPPORTS
```

Comment:
Since you're removed `Protobuf` from device runtime, maybe it is to remove also this.
```suggestion
            depends on BR2_PACKAGE_HOST_PROTOBUF_ARCH_SUPPORTS
```